sections/dashboard/MobileDashboard/FuturesMarkets.tsx

```diff
@@ -14,17 +14,30 @@ const FuturesMarkets = () => {
 	const { t } = useTranslation();
 
 	const futuresMarkets = useRecoilValue(futuresMarketsState);
-
-	const dailyTradeStats = useGetFuturesDailyTradeStats();
+	const futuresVolumes = useRecoilValue(futuresVolumesState);
 
 	const openInterest = useMemo(() => {
 		return (
-			futuresMarkets
-				.map((market) => market.marketSize.mul(market.price).toNumber())
-				.reduce((total, openInterest) => total + openInterest, 0) ?? null
+			futuresMarkets.reduce(
+				(total, { openInterest }) =>
+					total.add(openInterest?.shortUSD ?? wei(0)).add(openInterest?.longUSD ?? wei(0)),
+				wei(0)
+			) ?? null
 		);
 	}, [futuresMarkets]);
 
+	const [trades, volume] = useMemo(() => {
+		const totalTrades = Object.values(futuresVolumes).reduce(
+			(total, { trades }) => total.add(trades),
+			wei(0)
+		);
+		const totalVolume = Object.values(futuresVolumes).reduce(
```

Comment:
Not a big deal but could be nice to combine these in a single reduce to save repeating the loop.

Reply:
good call, I combined them